.github/workflows/testing.yml

```diff
@@ -355,3 +355,28 @@ jobs:
       - name: Itemized validation
         if: ${{ !cancelled() }}
         run: bin/test_itemcheck
+
+  discoverynode:
+    name: Discovery Node
+    if: ${{ !cancelled() && vars.TARGET_PROJECT != '' }}
+    needs: [ baseline, sequencer, redirect, runlocal ]
```

Comment:
should get this running local-only for now, to prevent making the overall CI test time too long. Alternatively we can fork out another parallel registry for this (and one of the other three).   (And, this needs: clause is unnecessarily broad -- figure out where it goes in the chain and adjust that accordingly)

For now, I'd be following the pattern of the runlocal: job (which always runs and only uses local testing)

Reply:
Updated to use local UDMIs for testing